<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateNewDatabase</class>
 <widget class="QDialog" name="CreateNewDatabase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>438</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>863</width>
    <height>438</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>863</width>
    <height>438</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tblCreateTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>821</width>
     <height>241</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>821</width>
     <height>241</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>821</width>
     <height>241</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>250</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnAddRow">
      <property name="text">
       <string>Add New Row</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDeleteRow">
      <property name="text">
       <string>Delete Last Row</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>251</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btnExecute">
      <property name="text">
       <string>Execute</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lbDbLocation">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>801</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New Database Location: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowseNewDb">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>371</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To completely prepare a new database you must also prepare it's first table.</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="lbNewTable">
      <property name="text">
       <string>New Table Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leTableName"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
